.github/workflows/forge-unstable.yaml

```diff
@@ -30,8 +30,11 @@ env:
   AWS_ACCOUNT_NUM: ${{ secrets.ENV_ECR_AWS_ACCOUNT_NUM }}
   AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
   AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-  IMAGE_TAG: ${{ inputs.IMAGE_TAG }} # this is only used for workflow_dispatch, otherwise defaults to empty
   AWS_REGION: us-west-2
+  GCP_WORKLOAD_IDENTITY_PROVIDER: ${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}
```

Comment:
Do you need this? 

Looks like GCP_WORKLOAD_IDENTITY_PROVIDER and other gcp auth vars are automatically pulled in in the workflow via secrets context

Reply:
Not sure. I'll remove them to see if the workflow works without.

Reply:
Done.